docs/extend/views.md

```diff
@@ -0,0 +1,67 @@
+# Views and Blade
+
+Although the Flarum UI you know and love is powered by our [Mithril frontend](frontend.md),
+server-side generated templates are still used throughout Flarum. Most notably, the HTML skeleton of the forum, which includes various SEO meta tags, as well as the no-js view of the forum, is implemented through the Views and Blade systems.
+
+[Blade](https://laravel.com/docs/8.x/blade) is Laravel's templating engine, which allows you to conveniently generate HTML (or other static content) from PHP.
+It's the same idea as [Jinja](https://jinja.palletsprojects.com/en/3.0.x/) or [EJS](https://ejs.co/).
+[Views](https://laravel.com/docs/8.x/views) are Laravel's system for organizing/registering Blade templates, and also includes utilities for rendering them and providing them with variables.
+
+For our purposes, views are directories containing `.blade.php` template files (possible contained in subdirectories).
+
+## Adding Views
+
+You will need to tell the view factory where it can find your extension's view files by adding a `View` extender to `extend.php`:
+
+```php
+use Flarum\Extend;
+use Illuminate\Contracts\View\Factory;
+
+return [
+    (new Extend\View)
+        ->namespace('acme.hello-world', __DIR__.'/views');
+];
+```
+
+
+## Blade Templates
+
+To learn about the syntax for Blade templates, read [Laravel's documentation](https://laravel.com/docs/8.x/blade).
+
+Once you've set up your views, you'll can render them to strings:
+
+```php
+// Here, $view is of type `Illuminate\Contracts\View\Factory`
+$renderedString = $view->make('acme.hello-world::greeting')->render();
+
+// You can also pass variables to the view:
+$renderedString = $view->make('acme.hello-world::greeting', ['varName' => true])->render();
+```
+
+You can obtain the view factory instance through dependency injection.
+
+The format is "VIEW_NAMESPACE::VIEW_NAME". If the view folder is organized as subdirectories, replace `/` with `.` in the pack.
```

Comment:
Lines 43 to 51, maybe use backquotes to render all namespaces and variable names as inline code in markdown?